test/js-native-api/test_constructor/test_constructor.c

```diff
@@ -4,10 +4,25 @@
 static double value_ = 1;
 static double static_value_ = 10;
 
+static void add_named_property(napi_env env, const char * key, napi_value return_value) {
```

Comment:
Nit: No space before `*`. I would call this function `add_last_error_message()`.